wwwapp/models.py

```diff
@@ -59,6 +59,47 @@ class WorkshopUserProfile(models.Model):
     def __str__(self):
         return '%s: %s, %s' % (self.year, self.user_profile, self.status)
 
+class PESELField(models.CharField):
```

Comment:
Maybe we should consider moving it into a separate file? Not sure though.

Reply:
This applies to most of the code base as well - this should be done in a seperate PR under the name "Refactor all the things".